Library/Homebrew/dev-cmd/audit.rb

```diff
@@ -532,8 +532,10 @@ def audit_conflicts
       end
     end
 
+    versioned_conflicts_whitelist = %w[node@ bash-completion@].freeze
```

Comment:
That sounds like a perfectly valid argument to me. When I get a chance, I will transform this into a true constant. The other lists should probably be done in a separate PR I think.